lib/x11utils.pm

```diff
@@ -511,7 +511,7 @@ Also handle workarounds when needed.
 =cut
 sub handle_welcome_screen {
     my (%args) = @_;
-    assert_screen([qw(opensuse-welcome opensuse-welcome-boo1169203 opensuse-welcome-gnome40-activities)], $args{timeout});
+    assert_screen([qw(opensuse-welcome opensuse-welcome-gnome40-activities)], $args{timeout});
     send_key 'esc' if match_has_tag('opensuse-welcome-gnome40-activities');
     workaround_broken_opensuse_welcome_window() if match_has_tag("opensuse-welcome-boo1169203");
```

Comment:
You removed `opensuse-welcome-boo1169203` above, so this is dead code

Reply:
Ah sorry I didn't understand you. My intention here was to avoid this wrong soft-fail imo : https://openqa.opensuse.org/tests/2255543#step/opensuse_welcome/1

Reply:
In the upgrade case it's definitely wrong, but is the referenced bug actually gone?

Reply:
As the bug issue has never been reported again since October of 2020, I don't find the reason that we should keep this workaround in the code.

Reply:
Agreed. Can you remove all the relevant code, preferably in a separate commit?